.github/release.yml

```diff
@@ -0,0 +1,23 @@
+changelog:
```

Comment:
How do you use this semantic versioning?

Reply:
do we need to use this: https://www.npmjs.com/package/semantic-release ?

Reply:
Nop, it's a github native changelog generation thing which generates entries based on labels in pull requests:
<img width="983" alt="image" src="https://user-images.githubusercontent.com/12693088/209006998-49238710-b5f2-458c-ae08-b7ebce904851.png">

It should automatically create a release with the changelog based on these rules via `.github/workflows/changelog.yml` when a tag is pushed.

All that needs to happen is triggering manual github action with desired version number:
<img width="347" alt="image" src="https://user-images.githubusercontent.com/12693088/209007415-6af91be1-e4f7-43c9-b8b0-cbc0b2b7b65e.png">

It can be improved where one could have options (major, minor, patch) instead of actual version, but that can be added later, no external packages necessary, just a few lines of ruby code.